Projects/Server/Json/Converters/ClientVersionConverter.cs

```diff
@@ -0,0 +1,37 @@
+/*************************************************************************
+ * ModernUO                                                              *
+ * Copyright 2019-2020 - ModernUO Development Team                       *
+ * Email: hi@modernuo.com                                                *
+ * File: MapConverter.cs                                                 *
```

Comment:
Let's update this to ClientVersionConverter.cs.
I wonder if VS has an autocomplete macro like Rider does. 🤔

Reply:
I use Rider haha can you show me how it works?

Reply:
@michaelwhelehan import the https://github.com/modernuo/ModernUO/blob/main/rider-settings.zip file into your settings.

Then you type `copy` and will see a dropdown for copyright, press enter, and it is done for you.